adapters/repos/db/shard_read.go

```diff
@@ -364,3 +364,71 @@ func (s *Shard) buildAllowList(ctx context.Context, filters *filters.LocalFilter
 
 	return list, nil
 }
+
+func (s *Shard) docIDToUUID(docID uint64) (strfmt.UUID, error) {
```

Comment:
I think that `uuidFromDocID` reads a bit better, but that is just my preference, this makes sense too

Reply:
yes, I agree too. done!